overrides/macos-override.json

```
@@ -549,7 +549,10 @@
             ]
         },
         "duckPlayer": {
-            "state": "enabled"
+            "state": "enabled",
+            "settings": {
+                "tryDuckPlayerLink": "https://www.youtube.com/watch?v=yKWIA-Pys4c",
```

```suggestion
                "tryDuckPlayerLink": "https://www.youtube.com/watch?v=yKWIA-Pys4c"
```
This comma needs to be dropped.